src/qt/rpcconsole.cpp

```diff
@@ -1200,9 +1204,9 @@ void RPCConsole::unbanSelectedNode()
         CSubNet possibleSubnet;
 
         LookupSubNet(strNode.toStdString().c_str(), possibleSubnet);
-        if (possibleSubnet.IsValid() && g_connman)
+        if (possibleSubnet.IsValid() && g_banman)
```

Comment:
Commit [banman: create and split out banman].

`g_banman` is guaranteed to exist given the check at the top of the function.

Reply:
Yep, will fix.